src/CMakeLists.txt

```diff
@@ -3,6 +3,27 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 include(${UMF_CMAKE_SOURCE_DIR}/cmake/helpers.cmake)
+        
+if(UMF_BUILD_LEVEL_ZERO_PROVIDER)
+    include(FetchContent)
+
+    set(LEVEL_ZERO_LOADER_REPO "https://github.com/oneapi-src/level-zero.git")
+    set(LEVEL_ZERO_LOADER_TAG v1.16.1)
+    
+    FetchContent_Declare(level-zero-loader
+        GIT_REPOSITORY    ${LEVEL_ZERO_LOADER_REPO}
+        GIT_TAG           ${LEVEL_ZERO_LOADER_TAG}
+        EXCLUDE_FROM_ALL
+    )
+
+    FetchContent_GetProperties(level-zero-loader)
+    if(NOT level-zero-loader_POPULATED)
+        FetchContent_Populate(level-zero-loader)
+    endif()
+
+    set(LEVEL_ZERO_INCLUDE_DIRS ${level-zero-loader_SOURCE_DIR}/include 
+        CACHE PATH "Path to Level Zero Headers")    
+endif()
```

Comment:
perhaps, pls print `LEVEL_ZERO_INCLUDE_DIRS` here (similarly to e.g. : https://github.com/oneapi-src/unified-runtime/blob/main/source/adapters/opencl/CMakeLists.txt#L81 )

Reply:
added, but note that we should do this probably for all external components

Reply:
done